concourse/steps/release.mako

```diff
@@ -70,12 +70,23 @@ release_commit_publishing_policy = release_trait.release_commit_publishing_polic
 if release_commit_publishing_policy is ReleaseCommitPublishingPolicy.TAG_ONLY:
   merge_back = False
   push_release_commit = False
+  create_release_commit = True
+  bump_commit = True
 elif release_commit_publishing_policy is ReleaseCommitPublishingPolicy.TAG_AND_PUSH_TO_BRANCH:
   merge_back = False
   push_release_commit = True
+  create_release_commit = True
+  bump_commit = True
 elif release_commit_publishing_policy is ReleaseCommitPublishingPolicy.TAG_AND_MERGE_BACK:
   push_release_commit = False
   merge_back = True
+  create_release_commit = True
+  bump_commit = True
+elif release_commit_publishing_policy is ReleaseCommitPublishingPolicy.SKIP:
+  push_release_commit = False
+  merge_back = False
+  create_release_commit = false
```

Comment:
nit: `false` -> `False`

Reply:
again: good catch. I wonder why this did not break my local test 🤔